x-pack/plugins/rule_registry/server/utils/create_get_summarized_alerts_fn.ts

```diff
@@ -23,13 +23,15 @@ import {
   QueryDslQueryContainer,
   SearchTotalHits,
 } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import { AlertsFilter } from '@kbn/alerting-plugin/common';
+import { AlertsFilter, IsoWeekday } from '@kbn/alerting-plugin/common';
 import { AlertHit, SummarizedAlerts } from '@kbn/alerting-plugin/server/types';
 import { ParsedTechnicalFields } from '../../common';
 import { ParsedExperimentalFields } from '../../common/parse_experimental_fields';
 import { IRuleDataClient, IRuleDataReader } from '../rule_data_client';
 
 const MAX_ALERT_DOCS_TO_RETURN = 100;
+export const ISO_WEEKDAYS: IsoWeekday[] = [1, 2, 3, 4, 5, 6, 7];
```

Comment:
Already exported from `@kbn/triggers-actions-ui-plugin/common/constants`, but probably should be exported from `alerting-plugin`. Can you move this there?